tests/test_discrete_problem.py

```diff
@@ -90,17 +97,15 @@ def test_aggregation_with_extreme_value_shocks(
         n_extra_axes,
     )
 
-    calculator = get_emax_calculator(
-        shock_type="extreme_value",
-        variable_info=var_info,
-        is_last_period=False,
+    choice_axes = _determine_dense_discrete_choice_axes(var_info)
+    solve_discrete_problem = partial(
+        _calculate_emax_extreme_value_shocks,
+        choice_axes=choice_axes,
```

Comment:
We do this manually until support for utility/taste shocks is added. Then we will change this back.